releases/net472/KnownIssues/695400 - .NET 4.7.2 setup skips installing payload on Win10 machine when upgraded from Win7 with .NET 4.7.2 .md

```diff
@@ -0,0 +1,42 @@
+# .NET 4.7.2 setup fails to install files on OS when upgraded from Windows 7 SP1 or Windows Server 2008 R2 with .NET Framework 4.7.2 .
+
+## Symptoms
+When you install the .NET Framework 4.7.2 on Windows 10 version 1803 or lower and your system was upgraded from Windows 7 SP1 with .NET Framework 4.7.2 , .NET 4.7.2 setup installation reports success but fails to install .NET 4.7.2 files in the machine.
+
+Consider the following scenario:
+  1. On a Windows 7 SP1 or Windows Server 2008 R2 , install the .NET Framework 4.7.2
+  2. Upgrade the machine to Windows 10 Fall Creators (version 1709) or lower. Upgraded OS can be any version of Windows 8.1 or Windows Server 2008 R2.
+  3. Run .NET 4.7.2 setup
+  4. .NET Framework 4.7.2 setup reports successful installation but fails to install framework files.
+
+Note: This issue does not occur when upgraded to Windows 10 April 2018 Update (version 1803) as it comes pre-installed with .NET Framework 4.7.2.
+
+## Cause
+This issue occurs because OS upgrade leaves behind .NET 4.7.2 setup file in Windows installer cache and its registration. Due to a bug in .NET setup that checks for the setup file in Windows installer and determines that .NET 4.7.2 is already installed and thus, skips installing .NET framework files on the upgraded Windows 10 machine.
+
+## Impact
+.NET 4.7.2 setup reports successful installation but skips installing its files and registries.
+
+Note: This issue occurs only when you do an upgrade from Windows 7 or Windows Server 2008 R2 with .NET 4.7.2 pre-installed.
+
+## Resolution
+Upgrade to latest Windows 10 April 2018 Update ( Version 1803), it comes pre-installed with .NET Framework 4.7.2 .
+
+Alternatively, you can install .NET Framework 4.7.2 on failure machine by directly installing the respective MSU from Windows Catalog
+* [Windows Server 2012](https://www.catalog.update.microsoft.com/Search.aspx?q=4054542)
+* [Windows 8.1 or Windows Server 2012 R2](https://www.catalog.update.microsoft.com/Search.aspx?q=4054566)
+* [Windows 10 Anniversary Update or Windows 10 Creator Update](https://www.catalog.update.microsoft.com/Search.aspx?q=4054590)
+* [Windows 10 Fall Creators ](https://www.catalog.update.microsoft.com/Search.aspx?q=4073120)
+
+
+If you choose not to follow above two resolutions, you can run below commands from an administrator command prompt to remove setup registration and re-run .NET Framework 4.7.2 setup
```

Comment:
- above two resolutions --> the two previous resolutions
- run below commands --> run the following commands
- remove setup --> remove the setup
- period after ".NET Framework 4.7.2 setup